docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,228 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
```

Comment:
@qnkhuat @calherries 

Could you please take another look at `convertTimezone`? I've made a lot of updates since last time. I think this doc is the most complex, so it should be a good catch-all for timestamp nuances in general (I'll follow up if I need to generalize the feedback to any other datetime docs).

By end of week would be amazing, but I think these docs can wait until next week too. Thank you!

Reply:
In general, I think this is much clearer. only have some nitty comments :)